dali/operators/reader/numpy_reader_op.cc

```diff
@@ -124,7 +124,8 @@ If ``file_root`` is provided, the paths are treated as being relative to it.
 This argument is mutually exclusive with ``file_list``.)", nullptr)
   .AddOptionalArg("register_buffers",
       R"code(Applies **only** to the ``gpu`` backend type.
-
+.. warning:
+this feature is temporarily disabled and will be re-enabled in future releases.
```

Comment:
```suggestion
  This argument is temporarily disabled and left for the backward compatibility.
  It will be reenabled in the future releases.
```

Reply:
I'd say "for backward compatibility" and "in future releases". No "the" in those, IMO.

Reply:
I can agree with the wording, but the format is not valid rst (and the sentence should start with the capital letter).

Reply:
Sure